src/Compilers/VisualBasic/Portable/Compilation/VisualBasicDiagnosticFilter.vb

```diff
@@ -54,6 +55,7 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             ' We don't permit configuring 1607 and independently configuring the new warnings.
 
             Dim report As ReportDiagnostic
+            Dim hasDisableDirectiveSuppression As Boolean = False
```

Comment:
minor: I think it may be better to just use "hasSourceSuppression" in both languages. "hasDisableDirectiveSuppression" reads a bit strange :)